### ISSUE TITLE
mobile Twitter peculiarities with Block Service Workers

### ISSUE THREAD

**Issue by grahamperrin** [clod81/block_service_workers#15]
Block Service Workers 1.0.0, Chromium 72.0.3626.121 (Official Build) (64-bit) on FreeBSD-CURRENT. 

tl;dr at https://mobile.twitter.com/settings/push_notifications after I click **NO** to block, I get none of these things: 

1. a block
2. the service worker at chrome://serviceworker-internals
3. push notification preferences at https://mobile.twitter.com/settings/push_notifications

![image](https://user-images.githubusercontent.com/192271/54571589-5ee14d80-49db-11e9-8765-26f9bfb44553.png)

----

If, instead, I click **YES** to allow then there's an apparent allowance but still, non-appearance of preferences: 

![image](https://user-images.githubusercontent.com/192271/54571855-7cfb7d80-49dc-11e9-90f1-0d1607a7688f.png)

I waited a long time for preferences to appear, they didn't. 

Quit, relaunched Chromium: no difference (still non-appearance of preferences within the Twitter window). 

A few seconds after I disabled the extension, preferences appeared: 

![image](https://user-images.githubusercontent.com/192271/54571987-f6936b80-49dc-11e9-906b-5a3f75e3daf7.png)

– and I gained a worker for mobile Twitter.

**Comment by clod81** [comment]
Sorry of the late reply.
I was able to reproduce it, yes. Unsure what would cause that, I'd need to dig into it.
As a workaround, like you mentioned, choose 'ALLOW', disable the extension, reload the page and finally re-enable the extension.
